lesson2/MainActivity.java/lib/src/main/java/com/example/Fraction.java

```diff
@@ -35,14 +35,12 @@ public Fraction simplify() {
         int numGCD = gcd(numerator, denominator);
         numerator = numerator/numGCD;
         denominator = denominator/numGCD;
-        Fraction simpleFrac= new Fraction(numerator,denominator);
-        return simpleFrac;
-
+        return new Fraction(numerator, denominator);
     }
 
     public Fraction add(Fraction f2) {
-        int numerator = this.numerator*f2.denominator+f2.numerator*denominator;
-        int denominator = this.denominator*f2.denominator;
+        int numerator = this.numerator * f2.denominator + f2.numerator * denominator;
+        int denominator = this.denominator * f2.denominator;
```

Comment:
Like the `==` above, it's also a good idea to put spaces on either side of math operations like `*` and `+`, makes your code more readable!